app/models/timer.rb

```diff
@@ -0,0 +1,31 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+  belongs_to :game
+
+  def initialize
```

Comment:
This is breaking the inheritance chain for the ActiveRecord superclass. You either need to call the super, or (better) use a `after_initialize` callback instead.